File: Solved/04344/04344.cpp
// Problem No.: 4344
// Solver:      Jinmin Goh
// Date:        20191128
// URL: https://www.acmicpc.net/problem/4344

#include <iostream>
#include <list>
#include <stdlib.h>
using namespace std;

int main()
{
    // this code makes input & output fast as scanf / printf
    // cin.tie(NULL);
    // ios_base::sync_with_stdio(false);
    // check following blog: https://starrykss.tistory.com/750

    int n, std_cnt;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        cin >> std_cnt;
        int score[std_cnt] = {}, score_sum = 0, high_std_cnt = 0;
        for(int j = 0; j < std_cnt; j++)
        {
            cin >> score[j];
            score_sum += score[j];
        }
        for(int j = 0; j < std_cnt; j++)
        {
            if(score[j] > score_sum / (double)std_cnt)
                high_std_cnt++;
        }
        cout << fixed;
        cout.precision(3);
        cout << high_std_cnt / (double)std_cnt * 100 << '%' << endl;
    }
    
    return 0;
}

File: Solved/04344/04344.py
# Problem No.: 4344
# Solver:      Jinmin Goh
# Date:        20191128
# URL: https://www.acmicpc.net/problem/4344

import sys

n = int(input())
for i in range(n):
    std_score = input().split()
    std_cnt = int(std_score.pop(0))
    high_std_cnt = 0
    std_sum = 0
    for j in range(std_cnt):
        std_score[j] = int(std_score[j])
    for j in range(std_cnt):
        if std_score[j] > sum(std_score) / std_cnt:
            high_std_cnt += 1
    print("%0.3f" % float(round(high_std_cnt / std_cnt * 100, 3)), end="")
    print('%')
        